app/views/dashboards/followers.html.erb

```diff
@@ -3,17 +3,15 @@
 <div id="index-container"
   data-params="<%= params.to_json(only: %i[which per_page]) %>"
   data-which="followers">
-  <main id="main-content">
     <header class="crayons-layout">
       <h1 class="p-2 pb-0 m:p-0 crayons-title">Dashboard &raquo; Followers</h1>
       <%= render "actions_mobile" %>
     </header>
 
-    <div class="crayons-layout crayons-layout--2-cols">
-      <aside class="crayons-layout__sidebar-left">
+    <main id="main-content" class="crayons-layout crayons-layout--2-cols">
+      <nav class="crayons-layout__sidebar-left">
         <%= render "actions" %>
-      </aside>
-
+      </nav>
```

Comment:
refactor (blocking) - the content returned from `<%= render "actions" %>` is already a `nav` element, and so this change is having the effect of creating two navs nested inside each other. I think we can safely change the one here to a `div`, since it's just a container element used for positioning/display

(same for the other instances below)

Reply:
Oh right. Should have noticed that :sweat_smile: